.github/workflows/holonix-cache.yml

```diff
@@ -54,11 +54,16 @@ jobs:
             ${{ format(matrix.target, matrix.platform) }}
 
           # Don't exit if this fails so we can clean up the profile
+          total_status=0
           for i in result-*; do
-            cachix push holochain-ci $i || true
+            if ! cachix push holochain-ci $i; then
+              total_status=$((total_status+1))
```

Comment:
The push yes, but actually the bit we're trying to be careful about here is the `rm` step after the loop. I don't think it was ever proven, but the concern was about leaving symlinks behind that would prevent the Nix gc from cleaning up.

Now I think that the actions runner will create a temporary directory for builds and clean it up after the build but I believe the thinking was that the cleanup isn't immediate. So we could keep symlinks to the Nix store that would keep large amounts of data from being removed. 

I don't feel like finding out right now whether that will happen, so we try to run the `rm` step. We don't have this issue with the new Holonix because we're running on a public runner.

Reply:
Ah, I see 💡